crates/icrate/Cargo.toml

```diff
@@ -87,6 +87,47 @@ required-features = [
 [features]
 default = ["std", "apple"]
 
+# Disable to link specific frameworks, which may not available on lower OS versions, or users don't want to load the frameworks at startup (dyld).
```

Comment:
Sorry for being so slow to respond on this PR :/

I think I've changed my mind again on how I want this: We should go back to the single feature, but it should be called `unstable-no-link`, to signify that it should generally only be enabled by binary crates since it may break your build (since library crates are allowed to start depending on a framework in a minor release, and then they suddenly wouldn't be linked any more).

Reply:
Done.